SConstruct

```diff
@@ -11,6 +10,8 @@ import glob
 import sys
 import methods
 
+from SCons.Script import BUILD_TARGETS
```

Comment:
BTW from what I could read in the scons docs, I think stuff from SCons.Script is already in scope in SConstruct.